### DISCUSSION TITLE
求助一下datastore_engine_options应该怎么写

### DISCUSSION THREAD

**Discussion by Tosd0** [he0119/nonebot-plugin-datastore#128] [Q&A] [answered]
遇见了sqlite的`sqlalchemy.exc.OperationalError: (sqlite3.OperationalError) database is locked`问题，看见需要改poolclass，但是配置文件实在是不会（跪）
配置里的datastore_engine_options，试了好几种方法都报错了😭
.env中写成`datastore_engine_options={"poolclass": "sqlalchemy.pool.StaticPool"}`的报错：
https://pastebin.com/yqKD2x6W
.env中写成`datastore_engine_options={"poolclass": sqlalchemy.pool.StaticPool}`或者`datastore_engine_options={"poolclass": pool.StaticPool}`或者`datastore_engine_options={"poolclass": StaticPool}`的报错：
https://pastebin.com/WyeL8F77

**Comment by he0119** [comment, marked as answer]
这种需要传递一个类的，你需要在 `bot.py` 里改。

<https://github.com/he0119/CoolQBot/blob/4a21d6297ff2ba394571977c89a943148cbfda21/bot.py#L7>
